packages/opencensus-core/src/stats/recorder.ts

```diff
@@ -40,6 +41,20 @@ export class Recorder {
     }
   }
 
+  /** Gets the tag values from tags and columns */
+  static getTagValues(tags: Map<TagKey, TagValue>, columns: TagKey[]):
```

Comment:
Why is this a `static` method of the class rather than an exported function in the file? Is that because the other similar functions are static methods of the class?

By default in TypeScript, I would prefer exported functions in a module rather than static methods of a class.

Reply:
> By default in TypeScript, I would prefer exported functions in a module rather than static methods of a class.

I agreed, but currently all other methods are static in the recorder class. I wanted to keep the same notion.